evap/staff/templates/staff_course_type_index.html

```diff
@@ -52,9 +52,8 @@
             </tbody>
         </table>
         <div class="form-group well">
-            <div class="col-sm-6">
-                <input type="submit" value="{% trans "Save course types" %}" class="btn btn-primary"/>
-            </div>
+            <input type="submit" value="{% trans "Save course types" %}" class="btn btn-primary"/>
+            <a href="{% url "staff:course_type_merge_selection" %}" class="btn btn-default">{% trans "Merge course types" %}</a>
```

Comment:
isn't that a bit weird, a button below a form that actually has nothing to do with the form?